libraries/AP_HAL_ChibiOS/GPIO.cpp

```diff
@@ -215,6 +215,17 @@ bool GPIO::usb_connected(void)
     return _usb_connected;
 }
 
+#ifdef HAL_GPIO_PIN_PWM_VOLT_SEL
```

Comment:
I don't think we need this change in HAL_ChibiOS, just use hal.gpio interface from AP_BoardConfig